qbittorrent.json

```
@@ -0,0 +1,64 @@
+{
+    "qBittorrent": {
+        "containers": {
+            "qbittorrent-linuxserver.io": {
+                "image": "linuxserver/qbittorrent",
+                "launch_order": 1,
+                "opts": [
+                    [
+                        "-e",
+                        "UMASK_SET=007"
+                    ],
+                    [
+                        "-e",
+                        "WEBUI_PORT=8090"
+                    ]
+                ],
+                "ports": {
+                    "8090": {
```

Set webUI port to an exotic number **hopefully** unused on most users' machine:

```suggestion
                    "12500": {
```